pkg/chunked/cache_linux_test.go

```diff
@@ -164,7 +167,8 @@ func TestWriteCache(t *testing.T) {
 			if digest != r.ChunkDigest {
 				t.Error("wrong digest found")
 			}
-			expectedLocation := generateFileLocation(r.Name, uint64(r.ChunkOffset), uint64(r.ChunkSize))
+			expectedLocation, err := generateFileLocation(0, uint64(r.ChunkOffset), uint64(r.ChunkSize))
```

Comment:
Non-blocking: Testing this with at least two distinct paths, to truly exercise the indexing logic, would be nice.